### DISCUSSION TITLE
MasterTemp Heater Not a Showing Up In Schedule

### DISCUSSION THREAD

**Discussion by kmontagne** [tagyoureit/nodejs-poolController#859] [General]
I have a MasterTemp heater defined in my Intellicenter. Everything works fine, except it does not show up as Heat Source option when editing a schedule. The heater shows up in the Intellicenter2 app when editing a schedule.
![image](https://github.com/tagyoureit/nodejs-poolController/assets/3219609/83e834fa-f9d2-4adb-9109-2c9c47c7b1f1)
![image](https://github.com/tagyoureit/nodejs-poolController/assets/3219609/1dbc3e17-3c42-40c4-b149-c452d4a1c2d8)
![image](https://github.com/tagyoureit/nodejs-poolController/assets/3219609/180fd87b-130e-47d6-8496-3d97bf9e7d25)

**Comment by tagyoureit** [comment]
Can you upload a [replay](https://github.com/tagyoureit/nodejs-poolController/wiki/How-to-capture-all-packets-for-issue-resolution#-dashpanel)?  It looks like we want to display a 3rd line in the drop down but it is showing up as a blank.
